### PR TITLE
Create a toaster to serve toast

### PR DESCRIPTION
**Pull request by nramadas**

### CODE REVIEW

File: src/views/components/toasters/ErrorToaster.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const T = React.PropTypes;
+
+export default class ErrorToaster extends React.Component {
```
Comment (nramadas) [inline src/views/components/toasters/ErrorToaster.jsx]: I'm likely going to consolidate the three toasters into one more generic component

Comment (curioussavage) [inline src/views/components/toasters/ErrorToaster.jsx]: :+1:

**Comment by nramadas** [comment]
:eyeglasses: @ajacksified or @schwers

File: assets/less/BodyLayout.less [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  &__toasterContainer {
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    max-width: 450px;
+    margin: 0 auto;
+    z-index: 11;
+
+
```
Comment (ajacksified) [inline assets/less/BodyLayout.less]: extra newline 😱

**Comment by nramadas** [comment]
:haircut:

**Comment by ajacksified** [comment]
:fish:

File: src/views/layouts/BodyLayout.jsx [hunk 3]
```diff
@@ -23,17 +26,28 @@ class BodyLayout extends BasePage {
...
     this._updateUserState = this._updateUserState.bind(this);
   }
 
   componentDidMount() {
     const app = this.props.app;
     app.on(constants.USER_DATA_CHANGED, this._updateUserState);
+    app.on(constants.TOASTER, this.toggleToaster);
+
+    // TODO: remove before deploying
+    window.test = type => {
```
Comment (ajacksified) [inline src/views/layouts/BodyLayout.jsx]: oh wait, remove this first

**Comment by ajacksified** [comment]
(pending moose)

**Comment by ajacksified** [comment]
Merged toasts - needs moose as a followup eventually.

**Pull request closed without merging**